appendix-basic65.tex

```diff
@@ -1940,6 +1945,11 @@ \subsection{COLOR}
                for screen characters. This speeds up screen
                handling, which could be useful when moving characters with different colours is
                not required.
+
+
+\item
+\item which is documented
+               on page \pageref{foregroundcommand}.
 \item [Example:] \screentext{COLOR ON} - with colour/attribute handling \\
```

Comment:
Maye we can add an example of the colour index usage too, e.g.:
```
\screentext{COLOR 2 : REM SET TEXT COLOUR TO RED}
```

Reply:
Aah, saw that you turned it into a program, so I tried the the program, and the 2nd line `COLOR OFF` had the side-effect of dismissing the 3rd line's attempt to change to red.

Maybe it'd be best to just have them like they were before, direct lines, but with explanations to the right, e.g.:
```
\screentext{COLOR 2} - set text colour to red \\
\screentext{COLOR ON} - with colour/attribute handling \\
\screentext{COLOR OFF} - no colour/attribute handling
```

Just for insight, here's a more elaborate example showing the COLOR ON/OFF behaviour:
![image](https://user-images.githubusercontent.com/20013309/140003007-27a1dc68-a130-402f-986f-05b13dbbe391.png)